scripts/cbw_pg_func.js

```diff
@@ -14,17 +14,143 @@ function addWatch() {
+                    '<tr><td>0.8 - 20</td><td><div class="surface_legend" style="background-color:#8000ff"></div></td></tr>' +
+                    '</tbody></table>',
+                fr: '<table><tbody><tr><th>valeur</th><th>couleur</th></tr>' +
+                    '<tr><td>-20 - -0.8</td><td><div class="surface_legend" style="background-color:#ff0000"></div></td></tr>' +
+                    '<tr><td>-0.8 - -0.6</td><td><div class="surface_legend" style="background-color:#008040"></div></td></tr>' +
+                    '<tr><td>-0.6 - -0.4</td><td><div class="surface_legend" style="background-color:#808040"></div></td></tr>' +
+                    '<tr><td>-0.4 - -0.2</td><td><div class="surface_legend" style="background-color:#ff8000"></div></td></tr>' +
+                    '<tr><td>-0.2 - -0.1</td><td><div class="surface_legend" style="background-color:#e8e800"></div></td></tr>' +
+                    '<tr><td>-0.1 - 0.1</td><td><div class="surface_legend" style="background-color:#c0c0c0"></div></td></tr>' +
+                    '<tr><td>0.1 - 0.2</td><td><div class="surface_legend" style="background-color:#00ffff"></div></td></tr>' +
+                    '<tr><td>0.2 - 0.4</td><td><div class="surface_legend" style="background-color:#0080ff"></div></td></tr>' +
+                    '<tr><td>0.4 - 0.6</td><td><div class="surface_legend" style="background-color:#0000ff"></div></td></tr>' +
+                    '<tr><td>0.6 - 0.8</td><td><div class="surface_legend" style="background-color:#ff00ff"></div></td></tr>' +
+                    '<tr><td>0.8 - 20</td><td><div class="surface_legend" style="background-color:#8000ff"></div></td></tr>' +
+                    '</tbody></table>'
+            }
+        }
+    ],
+
+    changeSurface: function(evt) {
+        let layerName = evt.target.value;
+        document.getElementById("surf_" + layerName).hidden = false;
+        document.getElementById("surf_" + avaIFaceJS.mapJS.cbw_func.currentSurface).hidden = true;
+        let legend = avaIFaceJS.cbw_func.surfaceLayers.find(r => r.name === layerName).legend[page_lang];
+        document.getElementById("surfLegend").innerHTML = legend;
+        avaIFaceJS.mapJS.cbw_func.changeSurface(layerName);
+    },
+
+    setOpacity: function(ev) {
+        let newOpacity = Math.max(10, ev.target.value) / 100;
+        avaIFaceJS.mapJS.cbw_func.wmts_layers.conformance.setOpacity(newOpacity);
+        avaIFaceJS.mapJS.cbw_func.wmts_layers.combined.setOpacity(newOpacity);
+        avaIFaceJS.mapJS.cbw_func.wmts_layers.difference.setOpacity(newOpacity);
+    },
 
     init: function() {
-        addWatch()
+        addWatch();
+        document.getElementById("surfTrans").addEventListener("input", avaIFaceJS.cbw_func.setOpacity);
+        document.getElementById("surfLegend").innerHTML = avaIFaceJS.cbw_func.surfaceLayers.find(r => r.name === "combined").legend[page_lang];
+        fetch("/dates.json", {method: "GET"})
```

Comment:
Should this be fetching from https://ava-proto.com/dates.json?

Reply:
This is true. Local development required a proxy and I mistakenly proxied this url. This should go to ava-proto.com

Reply:
> Is there any way we can across the scroll wheel zooming in and out while having the browser scrollbar remain visible. Having the map shifting side to side on entry and exit is kind of jarring.

I agree, but it was the only way I could get the scroll functionality to work. Working with Event Capturing/Bubbling didn't work as the event was never triggered when over the iframe. I can keep trying other options if this isn't sufficient.

Reply:
> > Is there any way we can across the scroll wheel zooming in and out while having the browser scrollbar remain visible. Having the map shifting side to side on entry and exit is kind of jarring.
> 
> I agree, but it was the only way I could get the scroll functionality to work. Working with Event Capturing/Bubbling didn't work as the event was never triggered when over the iframe. I can keep trying other options if this isn't sufficient.

Cool cool, just thought I'd double-check.  Let's leave it as is for now.

Reply:
I've just updated the request to point to ava-proto.com